### ISSUE TITLE
[Bug]: unable to use Coolify - ERROR The [public/storage] link already exists

### ISSUE THREAD

**Issue by venkadeshshiva** [coollabsio/coolify#2701]
### Description

unable to use Coolify. Coolify container restarting with error. 

### Minimal Reproduction (if possible, example repository)

install and run 

### Exception or Error

To support Server Side Up projects visit:
https://serversideup.net/sponsor
-------------------------------------
GID/UID
-------------------------------------

User uid:    9999
User gid:    9999
-------------------------------------

🏃‍♂️ Checking for Laravel automations...
🔐 Linking the storage...
🔓 SSL_MODE has been DISABLED, setting the web server to work in HTTP only...

   ERROR  The [public/storage] link already exists.  

In PhpRedisConnector.php line 159:
                    
  No route to host  
                    

s6-rc: warning: unable to start service db-migration: command exited 1
[27-Jun-2024 07:41:55] NOTICE: fpm is running, pid 100
[27-Jun-2024 07:41:55] NOTICE: ready to handle connections
[27-Jun-2024 07:41:55] NOTICE: systemd monitor interval set to 10000ms
/run/s6/basedir/scripts/rc.init: warning: s6-rc failed to properly bring all the services up! Check your logs (in /run/uncaught-logs/current if you have in-container logging) for more information.
prog: fatal: stopping the container.
[27-Jun-2024 07:41:56] NOTICE: Terminating ...
[27-Jun-2024 07:41:56] NOTICE: exiting, bye-bye!

### Version

4.0.0-beta.306

**Comment by coodyme** [comment]
same error here

**Comment by egorguscha** [comment]
its happening in all versions, I cant setup coolify

**Comment by venkadeshshiva** [comment]
if i disable firewall it works property. I don't know what is happening in behind the scene.

**Comment by nikoksr** [comment]
I encountered a similar issue related to the way filtering was handled on my host system. I installed Docker on a default Ubuntu server, which comes with **nftables** enabled by default. Docker does not natively support **nftables** ([see here](https://github.com/docker/for-linux/issues/1472)), as its network stack was developed when **iptables** were the default firewall solution for Linux. **iptables** are the predecessor of **nftables**.

In my case, the following line in my `nftables.conf` caused the problem:

```text
flush ruleset
```

This command flushes all previously set rules to ensure reproducible and consistent behavior. However, it also removes any rules set by Docker, which are necessary for container communication. As a result, containers fail to communicate with each other.

There are a few solutions to this problem. One that worked for me was replacing Docker with `podman` and `podman-docker`, as Podman natively supports **nftables**. Another potential solution is described [here](https://github.com/docker/for-linux/issues/1472#issuecomment-2189535038), or simply switching to an OS that doesn't use **nftables** by default.
